composer.json

```diff
@@ -100,7 +101,9 @@
         "webfontkit/open-sans": "^1.0",
         "webmozart/glob": "^4.7",
         "wikimedia/composer-merge-plugin": "2.1.0",
-        "yajra/laravel-pdo-via-oci8": "3.4.4"
+        "yajra/laravel-pdo-via-oci8": "3.4.4",
+        "league/commonmark": "2.3.9",
+        "elgigi/commonmark-emoji": "2.0.0"
```

Comment:
I think these lines should be removed -- they seem to be duplicates of dependencies already listed above (and in the case of league/commonmark, an older version).